azure-pipelines.yml

```diff
@@ -30,12 +30,12 @@ steps:
 
 - bash: |
     echo ">>> Compile checker-framework-vscode"
-    yarn && yarn compile
-    # Don't have the echo below, as it masks a failure from the yarn commands.
+    npm && npm compile
```

Comment:
The call to `npm` without an argument doesn't seem to do the same as it did with `yarn`: https://dev.azure.com/wmdietl/checker-framework-vscode/_build/results?buildId=443&view=logs&j=43d32c17-ff19-5d0a-58b3-52a09bf40d57&t=192742a1-5cf8-5181-5ceb-6819aba4bf5f&l=13
Should those invocations be removed?

Reply:
Yes, you are right. I just realized `npm` should use `npm run` + `scripts`.
Also, I should use `npm install` to set up the environment